aesara/link/numba/dispatch/elemwise.py

```diff
@@ -25,6 +27,10 @@
 from aesara.tensor.type import tensor
 
 
+numba_njit = partial(numba.njit, cache=config.numba__cache)
+numba_vectorize = partial(numba.vectorize, cache=config.numba__cache)
```

Comment:
This approach will require distinct `mock.patch`es for every module that creates these (e.g. `mock.patch("aesara.link.numba.dispatch.*.[numba_njit, numba_vectorize]", ...)`).  Instead, if you use something like `import aesara.link.numba.dispatch.basic as numba_basic` and `numba_basic.numba_njit` everywhere, all you'll need is a single mock for `aesara.link.numba.dispatch.basic.numba_jit`.

`mock`'s ability to patch an environment is fairly constrained, and not all import and access paths to patched objects are covered, so it's best to simplify the patched function import and access to a single, simple import from one module.